VTUL/dlp-access
# Change assets and data_url to file uploads

**Change assets and data_url to file uploads.**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-2393) (:star:)

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)

# What does this Pull Request do? (:star:)
Change assets and data_url to file uploads.

# What's the changes? (:star:)
* Change assets input field (text) to file upload. Display current value above file input.
* Change data_url input field (text) to file upload. Display current value above file input.

# How should this be tested?
* Visit Site Pages Config tab on /siteAdmin page
* Upload a test document in one of the "Assets" file inputs (there is one on each of the two current pages).
* Upload an html document in one of the "Data URL" file inputs (there is one on each of the two current pages).
* Submit the form and check that the changes are saved.

# Additional Notes:
* branch: `LIBTD-2393`

# Interested parties
@yinlinchen 

(:star:) Required fields
